Integration langfuse, front-end error

### Self Checks

- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [x] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [x] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [x] Please do not modify this template :) and fill in all the required fields.

### Dify version

1.0.1

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

After configuring langfuse in the application monitoring module, the front end reports an error:
Application error:a client-side exception has occurred (see the browser console for more information).
## langfuse detail
I tested both local and cloud langfuse and had the same problem

#### cloud langfuse
https://cloud.langfuse.com/
v3.41.0

#### local langfuse
v3.34.1 oss


## dify front-end  console log
```bash
8597-3349f9d5ea3c44e6.js:1 Error: Minified React error #130; visit https://react.dev/errors/130?args[]=null&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at iz (92df3e12-2414dde15c3f3d5d.js:1:113023)
    at d (92df3e12-2414dde15c3f3d5d.js:1:34852)
    at 92df3e12-2414dde15c3f3d5d.js:1:37512
    at u (92df3e12-2414dde15c3f3d5d.js:1:37803)
    at 92df3e12-2414dde15c3f3d5d.js:1:38870
    at lD (92df3e12-2414dde15c3f3d5d.js:1:57333)
    at iZ (92df3e12-2414dde15c3f3d5d.js:1:121360)
    at ia (92df3e12-2414dde15c3f3d5d.js:1:95166)
    at 92df3e12-2414dde15c3f3d5d.js:1:94988
    at il (92df3e12-2414dde15c3f3d5d.js:1:94995)
    at oJ (92df3e12-2414dde15c3f3d5d.js:1:92351)
    at oZ (92df3e12-2414dde15c3f3d5d.js:1:91770)
    at MessagePort.T (8597-3349f9d5ea3c44e6.js:1:84286)
push.84668.window.console.error @ 8597-3349f9d5ea3c44e6.js:1
l_ @ 92df3e12-2414dde15c3f3d5d.js:1
n.callback @ 92df3e12-2414dde15c3f3d5d.js:1
nB @ 92df3e12-2414dde15c3f3d5d.js:1
nV @ 92df3e12-2414dde15c3f3d5d.js:1
aq @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
a9 @ 92df3e12-2414dde15c3f3d5d.js:1
aY @ 92df3e12-2414dde15c3f3d5d.js:1
（匿名） @ 92df3e12-2414dde15c3f3d5d.js:1
is @ 92df3e12-2414dde15c3f3d5d.js:1
o1 @ 92df3e12-2414dde15c3f3d5d.js:1
oZ @ 92df3e12-2414dde15c3f3d5d.js:1
T @ 8597-3349f9d5ea3c44e6.js:1
3451-f3b01e497296e6f6.js:1 
            
            
           POST http://192.168.1.208:49604/console/api/apps/4ad7c160-fbbd-47eb-b3af-675673c18f4e/trace 500 (INTERNAL SERVER ERROR)
（匿名） @ 3451-f3b01e497296e6f6.js:1
T @ 3451-f3b01e497296e6f6.js:1
_fetch @ 3451-f3b01e497296e6f6.js:1
await in _fetch
s @ 3451-f3b01e497296e6f6.js:1
await in s
create @ 3451-f3b01e497296e6f6.js:1
e @ 3451-f3b01e497296e6f6.js:1
I @ 818-34a986c4f4ea323d.js:1
w @ 818-34a986c4f4ea323d.js:1
j @ 818-34a986c4f4ea323d.js:1
U @ 2952-f377b9d6306612c7.js:1
g @ page-06776ccb50aaf4b1.js:1
b @ page-06776ccb50aaf4b1.js:1
（匿名） @ page-06776ccb50aaf4b1.js:1
await in （匿名）
a_ @ 92df3e12-2414dde15c3f3d5d.js:1
aR @ 92df3e12-2414dde15c3f3d5d.js:1
（匿名） @ 92df3e12-2414dde15c3f3d5d.js:1
sF @ 92df3e12-2414dde15c3f3d5d.js:1
sM @ 92df3e12-2414dde15c3f3d5d.js:1
（匿名） @ 92df3e12-2414dde15c3f3d5d.js:1
o4 @ 92df3e12-2414dde15c3f3d5d.js:1
iV @ 92df3e12-2414dde15c3f3d5d.js:1
sU @ 92df3e12-2414dde15c3f3d5d.js:1
uR @ 92df3e12-2414dde15c3f3d5d.js:1
uM @ 92df3e12-2414dde15c3f3d5d.js:1
page-06776ccb50aaf4b1.js:1 Uncaught (in promise) Response {type: 'basic', url: 'http://192.168.1.208:49604/console/api/apps/4ad7c160-fbbd-47eb-b3af-675673c18f4e/trace', redirected: false, status: 500, ok: false, …}body: （…）bodyUsed: falseheaders: Headers {}ok: falseredirected: falsestatus: 500statusText: "INTERNAL SERVER ERROR"type: "basic"url: "http://192.168.1.208:49604/console/api/apps/4ad7c160-fbbd-47eb-b3af-675673c18f4e/trace"[[Prototype]]: Response
g @ page-06776ccb50aaf4b1.js:1
await in g
b @ page-06776ccb50aaf4b1.js:1
（匿名） @ page-06776ccb50aaf4b1.js:1
await in （匿名）
a_ @ 92df3e12-2414dde15c3f3d5d.js:1
aR @ 92df3e12-2414dde15c3f3d5d.js:1
（匿名） @ 92df3e12-2414dde15c3f3d5d.js:1
sF @ 92df3e12-2414dde15c3f3d5d.js:1
sM @ 92df3e12-2414dde15c3f3d5d.js:1
（匿名） @ 92df3e12-2414dde15c3f3d5d.js:1
o4 @ 92df3e12-2414dde15c3f3d5d.js:1
iV @ 92df3e12-2414dde15c3f3d5d.js:1
sU @ 92df3e12-2414dde15c3f3d5d.js:1
uR @ 92df3e12-2414dde15c3f3d5d.js:1
uM @ 92df3e12-2414dde15c3f3d5d.js:1
```

![Image](https://github.com/user-attachments/assets/a8765b41-4c0f-4432-8b62-10299b072679)

### ✔️ Expected Behavior

langfuse runs properly

### ❌ Actual Behavior

bug

--- a/web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/tracing/panel.tsx
+++ b/web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/tracing/panel.tsx
@@ -197,7 +197,7 @@ const Panel: FC = () => {
                 {t(`${I18N_PREFIX}.${enabled ? 'enabled' : 'disabled'}`)}
               </div>
             </div>
-            <InUseProviderIcon className='ml-1 h-4' />
+            {InUseProviderIcon && <InUseProviderIcon className='ml-1 h-4' />}
             <Divider type='vertical' className='h-3.5' />
             <div className='flex items-center' onClick={e => e.stopPropagation()}>
               <ConfigButton

